native/src/components/MapView.tsx

```diff
@@ -13,12 +13,27 @@ const StyledMap = styled(MapboxGL.MapView)`
   height: 500px;
 `
 
+type MapViewPropsType = {
+  boundingBox: BBox
+}
+
 // Has to be set even if we use map libre
 MapboxGL.setAccessToken(mapConfig.accessToken)
-const MapView: React.FunctionComponent = (): ReactElement => {
+const MapView = ({ boundingBox }: MapViewPropsType): ReactElement => {
+  const bounds = {
+    ne: [boundingBox[0], boundingBox[1]],
```

Comment:
i think the first two values standing for the minPoint (sw) the following for maxPoint(ne)

Reply:
jep. you're absolutely right. Fixed it.